grain_size_tools/__init__.py

```diff
@@ -0,0 +1,7 @@
+from . import piezometers
+from . import plots
+from . import tools
+from .GrainSizeTools_script import (
```

Comment:
based on your getting started instructions, it sounds like you would like `extract_column` and the other methods in the GrainSizeTools_script to be able to be imported top-level. By including this in the `__init__`, you will be able to do 
```
from grain_size_tools import extract_column
``` 
otherwise, you would need to do
```
from grain_size_tools.GrainSizeTools_script import extract_column
```